src/main/java/io/a2a/spec/JSONRPCResponse.java

```diff
@@ -13,8 +13,9 @@
  */
 @JsonInclude(JsonInclude.Include.NON_ABSENT)
 @JsonIgnoreProperties(ignoreUnknown = true)
-public sealed class JSONRPCResponse implements JSONRPCMessage permits SendTaskResponse, SendTaskStreamingResponse,
-        GetTaskResponse, CancelTaskResponse, SetTaskPushNotificationResponse, GetTaskPushNotificationResponse {
```

Comment:
does it make sense to be non-abstract?
Can we have a JSONRPCResponse which is none of the permitted subclasses?

Reply:
I'll do the same here.